src/cpp/rtps/builtin/discovery/participant/PDPClient.cpp

```diff
@@ -168,6 +168,8 @@ bool PDPClient::initPDP(RTPSParticipantImpl* part)
         return false;
     }
 
+    enable();
```

Comment:
Shouldn't this be done after initialization of WLP?

Reply:
I agree, changed.